sympy/polys/factortools.py

```diff
@@ -1301,12 +1303,14 @@ def dmp_factor_list(f, u, K0):
             if K0_inexact is None:
                 coeff = coeff/denom
             else:
+                num_factors = 0
                 for i, (f, k) in enumerate(factors):
                     f = dmp_quo_ground(f, denom, u, K0)
```

Comment:
It looks like some kind of normalization is necessary. One possibility might be setting the norm of each polynomial factor equal to one.